pkgs/misc/vere/default.nix

```
@@ -0,0 +1,48 @@
+{ stdenv
+, lib
+, fetchzip
+, writeScript
+}:
+
+let
+  os = if stdenv.isDarwin then "macos" else "linux";
+  arch = if stdenv.isAarch64 then "aarch64" else "x86_64";
+  platform = "${os}-${arch}";
+in
+stdenv.mkDerivation rec {
+  pname = "vere";
+  version = "1.20";
+
+  src = fetchzip {
+    url = "https://github.com/urbit/vere/releases/download/vere-v${version}/${platform}.tgz";
+    sha256 = {
+      x86_64-linux = "sha256-nBIpf9akK4cXnR5y5Fcl1g7/FxL8BU/CH/WHGhYuP74=";
+      aarch64-linux = "sha256-ERSYXNh/vmAKr4PNonOxTm5/FRLNDWwHSHM6fIeY4Nc=";
+      x86_64-darwin = "sha256-Kk9hNzyWngnyqlyQ9hILFM81WVw1ZYimMj4K3ENtifE=";
+      aarch64-darwin = "sha256-i3ixj04J/fcb396ncINLF8eYw1mpFCYeIM3f74K6tqY=";
+    }.${stdenv.hostPlatform.system} or (throw "unsupported system ${stdenv.hostPlatform.system}");
+  };
+
+  installPhase = ''
+    install -m755 -D vere-v${version}-${platform} $out/bin/urbit
+  '';
+
+  meta = with lib; {
+    name = "vere";
+    homepage = "https://urbit.org";
+    description = "An operating function";
+    platforms = [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin"];
+    maintainers = [ maintainers.matthew-levan ];
+    license = licenses.mit;
+  };
+
+  passthru.updateScript = writeScript "update-urbit" ''
+    #!/usr/bin/env nix-shell
+    #!nix-shell -i bash -p curl common-updater-scripts
+
+    set -eu -o pipefail
+
+    version="$(curl https://bootstrap.urbit.org/vere/live/last)
```

```suggestion
    version="$(curl https://bootstrap.urbit.org/vere/live/last)"
```

missing quote there. You can test the updateScript by running this in the nixpkgs root dir:

```
nix-shell maintainers/scripts/update.nix --argstr package urbit
```

Another problem I see is this script does not update all the system hashes.

I think this one for vault-bin would be a good one to copy: 
https://github.com/NixOS/nixpkgs/blob/5ab8b5ae26e6a4b781bdebdfd131c054f0b96e70/pkgs/tools/security/vault/vault-bin.nix#L52

---

Thanks again-- how's this look? I tested the update script, works on my machine (and it should on yours too, thanks to `nix` :).